src/dataset-builder/ConceptSelector.ts

```diff
@@ -103,10 +103,13 @@ export const ConceptSelector = (props: ConceptSelectorProps) => {
       }),
     ]),
     cart.length !== 0 &&
-      h(ActionBar, {
-        prompt: cart.length === 1 ? '1 concept selected' : `${cart.length} concepts selected`,
-        actionText,
-        onClick: () => _.flow(onCommit)(cart),
-      }),
+      h(Fragment, [
```

Comment:
Would it be worth it / possible to refactor to avoid code duplication?

Reply:
So, I put my thoughts on that here: https://github.com/DataBiosphere/terra-ui/pull/4918#discussion_r1657728538

I am not sure. It is not really great, I think, because it would attach positioning matters to the action bar.